src/utils/index.js

```diff
@@ -0,0 +1,39 @@
+// This will take an object and for each (k,v) will return
+// v rounded such that the sum of all v is 100.
+export const roundObjectPercentages = dataMap => {
+  // This algorithm workson integers and we want 2 decimal places.
+  // So round up first.
+  let asArray = Object.entries(dataMap).map(keyValue => {
```

Comment:
Maybe something like this would be more readable?

```suggestion
  let asArray = Object.entries(dataMap).map(([key, value])=> {
```

Reply:
Good one!